python/example_robot_data/robots_loader.py

```diff
@@ -3,21 +3,23 @@
 from os.path import dirname, exists, join
 
 import numpy as np
-
 import pinocchio as pin
 from pinocchio.robot_wrapper import RobotWrapper
 
 pin.switchToNumpyArray()
```

Comment:
This is not connected with your PR, however, we could remove this line.

Reply:
Yes, sorry, my isort seems not to be exactly configured in the same way on my different computers, which sometimes lead to this kind of useless changes. I noticed this one after the PR opening, and was too lazy to remove it, as it really doesn't harm.